src/sentry/api/endpoints/project_rule_actions.py

```diff
@@ -0,0 +1,36 @@
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry import eventstore
+from sentry.api.base import region_silo_endpoint
+from sentry.api.bases.rule import RuleEndpoint
+from sentry.rules.processor import RuleProcessor
+from sentry.utils.safe import safe_execute
+from sentry.web.decorators import transaction_start
+
+
+@region_silo_endpoint
+class ProjectRuleActionsEndpoint(RuleEndpoint):
+    private = True
+
+    @transaction_start("ProjectRuleActionsEndpoint")
+    def post(self, request: Request, project, rule) -> Response:
+        """
+        Activate a rule's downstream actions using the data from a previous event
+        """
+        events = eventstore.get_events(eventstore.Filter(project_ids=[project.id]), limit=1)
+        # TODO: use dummy event if there are no events in the project
+        if not events:
+            return Response(
+                "There are no events under this project that can be used to test", status=400
+            )
+        test_event = events[0]
+        # TODO: edit the event in a way to show that this is a test and not real
```

Comment:
Haven't really found a good way to do this yet, since I'm guessing events are supposed to be immutable? 

Also would be helpful if I could see how some of the integration notifications look locally, do you know if there's a way to do that?

Reply:
Email might work out of the box...not sure. You can also set up legacy webhooks and use [webhook.site](webhook.site)

Reply:
I know for slack we have a separate workspace + integration set up already you could configure. Haven't done it myself though

Reply:
@ceorourke might be able to point you in the direction of docs for setting up slack locally.

Reply:
https://develop.sentry.dev/integrations/slack/ it's kind of a pain though

Reply:
> https://develop.sentry.dev/integrations/slack/ it's kind of a pain though, lmk if you wanna be added to our shared workspace or you can use any workspace you already have.

thanks! could I get added to the shared workspace?